ditto/readers/cyme/read.py

```diff
@@ -4674,12 +4674,12 @@ def parse_transformers(self, model):
                 X0 = Z0 / math.sqrt(1 + 1 / (XR0 * XR0))
                 complex0 = complex(R0, X0)
                 complex1 = complex(R1, X1)
-                matrix = np.matrix(
+                matrix = np.array(
```

Comment:
Why was this change needed? `numpy` warnings?

Reply:
Yes.  They are planning to remove `Matrix`.